mkdocs.yml

```diff
@@ -41,6 +41,7 @@ nav:
     - Version Range Upgrades: howto/how-to-version-range-upgrades.md
     - Z-Stream Upgrades: howto/how-to-z-stream-upgrades.md
     - Derive Service Account Permissions: howto/derive-service-account.md
+    - Grant API Access: howto/how-to-grant-api-access.md
```

Comment:
I don't have a _great_ suggestion, but the "Grant API Access" title is a little ambiguous.

To a newcomer: Is it access to ClusterExtensions and ClusterCatalogs? Or something else? 

Maybe "Grant Access to Extension-Provided APIs" or something along those lines?

Reply:
WDYT of "Granting access to an API provided by a cluster extension"?

Reply:
Going with "Grant Access to Cluster Extension API". Keeps verb tense in line with the rest of them.